scenarios/05_Unexpected_Guests.cfg

```diff
@@ -164,13 +164,16 @@
         side=1
         shroud=no
         fog=yes
-        {GOLD 100 90 80}
+        {GOLD4 100 90 80 70}
         recruit=Akladian Wiseman,Akladian Clansman,Akladian Warrior
         [modifications]
             {TRAIT_LOYAL}
 #ifdef EASY
             {TRAIT_STRONG}
 #endif
+#ifdef NIGHTMARE
```

Comment:
Let's not debuff the player's units.

Reply:
[086c1d2](https://github.com/nemaara/A_New_Order/pull/53/commits/086c1d25c13ea2fc5549ae746b68eb2476cc283a).